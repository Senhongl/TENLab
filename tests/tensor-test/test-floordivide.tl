a = [[1.1,2.0],[3.5,4.2],[3.9, 0.6]] // 2.0;
print(a);