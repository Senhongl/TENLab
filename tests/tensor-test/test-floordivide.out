 0  1
 1  2
 1  0
[ CPUDoubleType{3,2} ]
